get_xpath.js

```diff
@@ -0,0 +1,71 @@
+// Taken with some modifications from - https://stackoverflow.com/questions/47069382/want-to-retrieve-xpath-of-given-webelement
+function absoluteXPath(element) {
+    var comp, comps = [];
```

Comment:
Use two character indentation for JS

Reply:
There are some problems with indentation in parts of the file (e.g. the indentation doesn't increase after `var getPos = function(element) {`).